cmake/onnxruntime.cmake

```diff
@@ -226,6 +226,13 @@ if (onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS)
   )
 endif()
 
+if (onnxruntime_USE_EXTENSIONS)
+  list(APPEND onnxruntime_INTERNAL_LIBRARIES
+    onnxruntime_extensions
```

Comment:
FYI, 'onnxruntime-extensions' is a cmake target alias, if it doesn't work iOS build, you may directly uses its original name.
https://github.com/microsoft/onnxruntime-extensions/blob/35408d94b6c1f6c720ad84e6f412fbcec039605d/CMakeLists.txt#L581

Reply:
thanks for the info. using this alias worked in my local iOS build so I think it should be good.